core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatterplot/ScatterplotOpDesc.java

```diff
@@ -65,9 +66,16 @@ public OperatorInfo operatorInfo() {
 
     @Override
     public Schema getOutputSchema(Schema[] schemas) {
-        return Schema.newBuilder().add(
-                new Attribute("xColumn", AttributeType.DOUBLE),
-                new Attribute("yColumn", AttributeType.DOUBLE)
-        ).build();
+        Schema inputSchema = schemas[0];
+        if(isGeometric)
```

Comment:
what is this change?

Reply:
I would recommend adding @sadeemsaleh to the review. She can review the visualization operators' changes as she has a better grip on them

Reply:
The `ScatterPlot` operator was stating a particular output schema in the `OperatorDescriptor`, but returning tuples of another schema from the opExec. I just moved that schema logic from the opExec to here. I will add @sadeemsaleh for this to verify. I verified the normal scatter plot using this change. Was not able to verify the scatter plot that renders on a map due to missing API key.

Reply:
this indeed shall be the case. looks good to me.